.clang-format

```diff
@@ -0,0 +1,102 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
+AccessModifierOffset: -1
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlines: Left
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: All
+AllowShortIfStatementsOnASingleLine: true
```

Comment:
Change to `false`? I have trouble following the flow of the program if there are conditionals on one  line. I realize it does take up some vertical space, but for whatever reason, it's harder for me to read code when it's not on a separate line.

The style guide (in the [conditional section](https://google.github.io/styleguide/cppguide.html#Conditionals)) states that they "may be written on one line if this enhances readability," which I read to mean that it should typically be one two lines instead of one. It doesn't say it has to be one way or another, though. There isn't a way to leave this as-is, right?